### DISCUSSION TITLE
Is there any update roadmap?

### DISCUSSION THREAD

**Discussion by wjdgus09** [styled-components/styled-components#4330] [Q&A] [answered]
Does anyone know if there is any document or content being discussed that provides information on the future update roadmap or direction of the styled-component library?

**Comment by quantizor** [comment, marked as answer]
There is no explicit roadmap. styled-components is in maintenance mode — the library will continue to receive bugfixes and minor improvements, but no major new features are planned.

For context, see the announcement: https://opencollective.com/styled-components/updates/thank-you
